lib/config.js

```diff
@@ -380,6 +385,7 @@ Config.prototype.getConfig = function (filePath) {
         config = util.mergeConfigs(config, this.useSpecificConfig);
     }
 
+    /* istanbul ignore else env is always set to either command line or empty object */
```

Comment:
This, for example. It's not possible to hit else branch. Ever. Since the code few lines above sets it to either user input or empty object, which will not pass this check.